### DISCUSSION TITLE
How to uninstall?

### DISCUSSION THREAD

**Discussion by dl8ndg** [nguyenquyhy/Flight-Tracker-StreamDeck#323] [Q&A]
What is the right procedure to uninstall Flight-Tracker-Stream-Deck?

-netbees-

**Comment by rmroc451** [comment]
Hello!

I _believe_, regardless of manually installing by putting it in the streamdeck plugin folder or installing from the store, you should be able to hit the "more actions" in the bottom right of your streamdeck UI, filter to **installed** and then scroll to find **Flight Tracker** and click uninstall.

![image](https://user-images.githubusercontent.com/15070945/148647521-7d16ba23-0e2b-428f-803f-296ae6fef025.png)

Alternatively, you could end the streamdeck.exe, delete the streamdeck folder here (%appdata%\Elgato\StreamDeck\Plugins\tech.flighttracker.streamdeck.sdPlugin), and restart the streamdeck software.
